sencha-workspace/SlateTasksTeacher/app/controller/Tasks.js

```diff
@@ -363,6 +363,12 @@ Ext.define('SlateTasksTeacher.controller.Tasks', {
                 Section: section.getData()
             }, task || null));
 
+            formPanel.getParentTaskField().getStore().filter({
+                id: 'exclude-self',
+                property: 'ID',
+                operator: '!=',
+                value: task.getId()
+            });
             formPanel.setTask(task);
```

Comment:
@nbey FYI, I also found it necessary that the filtering be updated BEFORE `setTask` loads the record into the form. Previously, if a parent task had been opened for editing immediately before the child task was, the parent task would be filtered out from the combo's values when the form loaded the child record and fail to match the existing ParentTaskID to the task record. This way, when the record is loaded, the available related values is already correct